test/e2e/network/netpol/kubemanager.go

```diff
@@ -60,13 +60,19 @@ func (k *kubeManager) initializeCluster(model *Model) error {
 
 		for _, pod := range ns.Pods {
 			framework.Logf("creating/updating pod %s/%s", ns.Name, pod.Name)
+			newPod := pod.KubePod()
+			if framework.NodeOSDistroIs("windows") {
```

Comment:
This logic should go in the model (that's the point of the model).

You can inject a boolean with `framework.NodeOSDistroIs("windows")` where the model is constructed, thread it through the model down to the pods, and use that to drive this logic inside of the `KubePod` method.